utility_network/display-utility-associations/src/main/java/com/esri/samples/display_utility_associations/DisplayUtilityAssociationsSample.java

```diff
@@ -0,0 +1,245 @@
+  
+  /**
+   * Opens and runs application.
+   *
+   * @param args arguments passed to this application
+   */
+  public static void main(String[] args) {
+    
+    Application.launch(args);
+  }
+  
+  @Override
+  public void start(Stage stage) {
+    
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+      
+      // set title, size, and add scene to stage
+      stage.setTitle("Display Utility Association Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+      
+      // create a map with the Topographic Vector Basemap
+      ArcGISMap map = new ArcGISMap(Basemap.createTopographicVector());
+      
+      // create a viewpoint from envelope
+      Viewpoint viewPoint = new Viewpoint(41.8057655, -88.1489692, 23);
+      
+      // create the utility network
+      utilityNetwork = new UtilityNetwork(FeatureServerUrl);
+      
+      // Load utility network async and get all of the edges and junctions in the network.
+      utilityNetwork.loadAsync();
+      utilityNetwork.addDoneLoadingListener(() -> {
+        List<UtilityNetworkSource> networkSources = utilityNetwork.getDefinition().getNetworkSources();
+        
+        for (UtilityNetworkSource nSource : networkSources)
+          // Check an add all edges that are not subnet lines to the map.
+          if (nSource.getSourceType() == UtilityNetworkSource.Type.EDGE && nSource.getSourceUsageType() != UtilityNetworkSource.UsageType.SUBNET_LINE) {
+            mapView.getMap().getOperationalLayers().add(new FeatureLayer(nSource.getFeatureTable()));
+          }
+          // Add all junctions to the map.
+          else if (nSource.getSourceType() == UtilityNetworkSource.Type.JUNCTION) {
+            mapView.getMap().getOperationalLayers().add(new FeatureLayer(nSource.getFeatureTable()));
+          }
+          
+        // add association graphics at the initial view point
+        addAssociation();
+        // listen for navigation changes
+        mapView.addNavigationChangedListener((event) -> {
+          addAssociation();
+        });
+      });
+
+      // create ImageView for teo symbol
+      ImageView attachmentImageView = new ImageView();
+      ImageView connectivityImageView = new ImageView();
+      
+      // Create a graphics overlay for associations.
+      associationsOverlay = new GraphicsOverlay();
+      
+      // Create Labels for the preview.
+      Label attachmentLabel = new Label("Attachment");
+      Label connectivityLabel = new Label("Connectivity");
+  
+      // Create Symbols for the associations and preview.
+      SimpleLineSymbol attachmentSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.DOT, 0xFF00FF00, 5);
+      SimpleLineSymbol connectivitySymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.DOT, 0xFFFF0000, 5);
+  
+      // get the symbol from the attachmentSymbol
```

Comment:
Let's try and be more specific here about what we're trying to achieve:

```suggestion
      // add the attachment symbol to the image view to display it in the legend
```